resources/r/model.R

```diff
@@ -62,10 +62,11 @@ manipulate_regions_for_shapefile <- function(region_prevalence) {
   
 #Add rank column/variable to dataset
 
-create_rank_variable_by_region <- function(thirteen_level_NHS_regional_prevalence){
```

Comment:
We need to update the flowchart so the functions/data names match - looks like the flowchart currently combines it with writing the narrative so we should split this out into a separate step in the flowchart